### PR TITLE
[Docs] Ensuring that all commands has usage examples

### LINKED ISSUE #8427: [Docs] Ensuring that all commands has usage examples
#### Desired State
Ensuring that all mesheryctl commands have at least usage example

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by suhail34**
**Notes for Reviewers**

This PR fixes #8427 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 655f9ec5e70bdf93952e2917aa1e6b8cdca2b2c3 at: https://64d93a55f606f94aa1fbc3ce--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8aa3ba4faa46322ded3a2fa8a4c9c602279ea963 at: https://64d9f4279f906a20158c2d80--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d202fb2a8cebafcbc63be1f8be1b64b11f41c488 at: https://64da40e2d2d5de4d8cd98345--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 477510a23d34ee838860b765240682b6329e9f40 at: https://64da4499aee17353078f0107--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c5793512db76da4127a1007f80fbb4f63c556394 at: https://64da60d2691a730759fedad8--meshery-docs-preview.netlify.app

File: mesheryctl/internal/cli/root/completion.go [hunk 1]
```diff
@@ -22,36 +22,36 @@ import (
...
 
-  # bash >= 4.0 on osx
-  brew install bash-completion@2
-  mesheryctl completion bash > $(brew --prefix)/etc/bash_completion.d/mesheryctl
+// bash >= 4.0 on osx
+brew install bash-completion@2
+mesheryctl completion bash > $(brew --prefix)/etc/bash_completion.d/mesheryctl
 
-  # zsh
+// ### zsh
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/completion.go:41]: Same here...

**Comment by suhail34** [comment]
> Also a small lady_beetle here... image
> 
> Once this is fixed, we're good

This is a issue for long time i am trying to figure out in this huge markdown file 🥲

**Comment by l5io** [comment]
🚀 Preview for commit aa72db0674470378796ca91f1c01677e5a0f1345 at: https://64dd29c31215dd0698c88424--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 56f349a1c51d3951ad57673d8bb0b5e23970f239 at: https://64de629e942d45009e98560f--meshery-docs-preview.netlify.app

File: mesheryctl/internal/cli/root/completion.go [hunk 2]
```diff
@@ -22,36 +22,36 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const example = `  # bash <= 3.2
-  source /dev/stdin <<< "$(mesheryctl completion bash)"
+const example = `
+// ### bash <= 3.2
```
[the remark this answers is not in the document]
Comment (alphaX86) [reply mesheryctl/internal/cli/root/completion.go:26]: Oh ok, then you can continue

**Comment by suhail34** [comment]
If there are some more changes to be done please suggest

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 3]
```diff
@@ -1,8 +1,7 @@
 global:
```
Comment (alphaX86) [inline docs/_data/mesheryctlcommands/cmds.yml:1]: Actually for this file only the name and description are fetched and not other key variables. But anyway, I'm not telling you to remove all though. So no issues here

**Review by alphaX86 — APPROVED** [review]

**Comment by alphaX86** [comment]
Fix the merge conflict and we're good @suhail34

**Comment by l5io** [comment]
🚀 Preview for commit 16b4e74a28b4836c4efe0d58114134c144cbd672 at: https://64e657faecae4d124e7d3ae5--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8abb2715a6cf70580656a743f08397954b11e62f at: https://64e666cac657880783f46fb5--meshery-docs-preview.netlify.app

**Comment by suhail34** [comment]
> Fix the merge conflict and we're good @suhail34

Done

**Comment by l5io** [comment]
🚀 Preview for commit c3362a2d40ab21d607f1f3567d2879cdf340a7c4 at: https://64e7a2a2403ecd095124212e--meshery-docs-preview.netlify.app

File: docs/pages/reference/mesheryctl-commands.md
Comment (alphaX86) [inline docs/pages/reference/mesheryctl-commands.md:1]: Only one change required, in line number 24

` 'mesheryctl filter' - Data Plane Intelligence: Registry and configuration of WebAssembly filters for Envoy (experimental feature, usage of 'exp' needed)`

Remove the usage of exp needed line. Rest all look good

Comment (suhail34) [reply docs/pages/reference/mesheryctl-commands.md:1]: Done @alphaX86

**Comment by l5io** [comment]
🚀 Preview for commit 416c5bf00838c305df99ed07a5383eaddb51fca7 at: https://64e86e91afe44452bc6afa9b--meshery-docs-preview.netlify.app

**Comment by alphaX86** [comment]
Thanks @suhail34 :)

**Pull request merged**